src/Tasks/SolrConfigureTask.php

```diff
@@ -4,27 +4,33 @@
 namespace Firesphere\SolrSearch\Tasks;
 
 use Exception;
+use Firesphere\SolrSearch\Helpers\SolrLogger;
 use Firesphere\SolrSearch\Indexes\BaseIndex;
 use Firesphere\SolrSearch\Interfaces\ConfigStore;
 use Firesphere\SolrSearch\Services\SolrCoreService;
-use Firesphere\SolrSearch\Stores\FileConfigStore;
-use Firesphere\SolrSearch\Stores\PostConfigStore;
+use Firesphere\SolrSearch\Stores;
 use Firesphere\SolrSearch\Traits\LoggerTrait;
-use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception\GuzzleException;
 use ReflectionException;
 use RuntimeException;
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\BuildTask;
+use SilverStripe\Dev\Debug;
+use SilverStripe\ORM\ValidationException;
 
+/**
+ * Class SolrConfigureTask
+ * @package Firesphere\SolrSearch\Tasks
+ */
 class SolrConfigureTask extends BuildTask
 {
     use LoggerTrait;
 
     protected static $storeModes = [
-        'file' => FileConfigStore::class,
-        'post' => PostConfigStore::class,
-//        'webdav' => WebdavConfigStore::class, // @todo
+        'file' => Stores\FileConfigStore::class,
```

Comment:
Adding the parent namespace, is this new programming practice?
Refactoring can also lead to moving a class from one namespace to another, hence more find and replace situation instead of top file only.

Reply:
It's using the parent namespace (see the `use` statements), reducing the amount of `use` statements in favour of using the parent inline.

I'm not happy with it either and I'm going to revert it. It's very "meh", not much more readable but a lot more clutter in the code itself.

Reply:
Reverted. A lot better this way 👍